boundary_layer_default_plugin/config/operators/hive_partition_sensor.yaml

```diff
@@ -0,0 +1,18 @@
+# This sensor detects hive partition changes
+name: hive_partition_sensor
+operator_class: HivePartitionSensor
+operator_class_module: airflow.sensors.hive_partition_sensor
+schema_extends: etsy_defaults_base
+parameters_jsonschema:
+    properties:
+        table:
+            type: string
+        partition:
+            type: string
+        metastore_conn_id:
+            type: string
```

Comment:
Hey Aldo, should we add these additional properties as well?

https://github.com/apache/airflow/blob/1.10.3/airflow/sensors/hive_partition_sensor.py#L51-L52